packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -118,6 +130,79 @@ void main() {
     );
   });
 
+  testWidgets('Stretch overscroll works in reverse - vertical', (WidgetTester tester) async {
+    final Key box1Key = UniqueKey();
+    final Key box2Key = UniqueKey();
+    final Key box3Key = UniqueKey();
+    final ScrollController controller = ScrollController();
+    await tester.pumpWidget(
+      buildTest(box1Key, box2Key, box3Key, controller, reverse: true),
+    );
+
+    expect(find.byType(StretchingOverscrollIndicator), findsOneWidget);
+    expect(find.byType(GlowingOverscrollIndicator), findsNothing);
+    final RenderBox box1 = tester.renderObject(find.byKey(box1Key));
+    final RenderBox box2 = tester.renderObject(find.byKey(box2Key));
+    final RenderBox box3 = tester.renderObject(find.byKey(box3Key));
+
+    expect(controller.offset, 0.0);
+    expect(box1.localToGlobal(Offset.zero), const Offset(0.0, 350.0));
+    expect(box2.localToGlobal(Offset.zero), const Offset(0.0, 100.0));
+    expect(box3.localToGlobal(Offset.zero), const Offset(0.0, -150.0));
+
+    final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(CustomScrollView)));
```

Comment:
This line seems to be exceeding 80 characters. I think we need to run `dartfmt` to re-format it a bit.

Reply:
The Flutter repository does not use dartfmt.